api/turing/cluster/servicebuilder/router.go

```diff
@@ -72,7 +72,7 @@ const (
 const (
 	defaultIstioGateway   = "istio-ingressgateway.istio-system.svc.cluster.local"
 	defaultGateway        = "knative-ingress-gateway.knative-serving"
-	defaultMatchURIPrefix = "/v1/predict"
+	defaultMatchURIPrefix = "/v1/predict,/v1/batch_predict"
```

Comment:
Would it be a better idea to define it as:
```go
var defaultMatchURIPrefixes = []string{"/v1/predict", "/v1/batch_predict"}
```
So you don't need to do the unnecessary string manipulations